packages/manager/src/features/Account/SwitchAccountDrawer.tsx

```diff
@@ -38,7 +38,8 @@ export const SwitchAccountDrawer = (props: Props) => {
   );
 
   const currentTokenWithBearer = useCurrentToken() ?? '';
-  const history = useHistory();
+  const currentParentToken =
```

Comment:
```suggestion
  const currentParentTokenWithBearer =
```
Optional for consistency